account_move_line_create_payment_entry/wizards/account_payment_entry_wizard.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, _, fields, models
+
+
+class AccountPaymentEntryWizard(models.TransientModel):
+    _name = "account.payment.entry.wizard"
+    _description = "Payment Entry Wizard"
+
+    account_id = fields.Many2one(
```

Comment:
Can we let users select a journal of the bank type instead?

Reply:
For the journal entry, I think we can only use the general type journal, isn't?